okhttp/src/commonJvmAndroid/kotlin/okhttp3/CompressionInterceptor.kt

```diff
@@ -98,7 +98,8 @@ open class CompressionInterceptor(
     /**
      * Request "gzip" compression.
      */
-    val Gzip =
+    @JvmField
+    public val Gzip =
```

Comment:
Make this a top-level object so Java people can do this?

```java
CompressionInterceptor interceptor = new CompressionInterceptor(
      Zstd.INSTANCE,
      Gzip.INSTANCE,
      Brotli.INSTANCE
)
```

Reply:
I tried it but `okhttp3.Gzip` felt weird.

Reply:
I'll follow up on this as I think it requires further discussion, and this is better.